### ISSUE TITLE
Central source for standard particle masses

### ISSUE THREAD

**Issue by kpedro88** [cms-sw/cmssw#29344]
A search for `m_pion` will find several slightly different values defined in different places: https://github.com/cms-sw/cmssw/search?q=m_pion

(this is probably not exhaustive, and there are others like kaons, etc.)

Ideally, for consistency all code should obtain these from a central source, much like other physical constants.

Not sure to what group this should be assigned...

**Comment by makortel** [comment]
assign reconstruction, generators, simulation

**Comment by fabiocos** [comment]
@kpedro88 @makortel in the past HepPDT was supposed to take exactly this role:

https://cmssdt.cern.ch/lxr/source/SimGeneral/HepPDTESSource/
https://cmssdt.cern.ch/lxr/source/SimGeneral/HepPDTRecord/

Most probably this is ignored by lot of people, who prefers to redefine autonomously the mass they need (not that different that redefining pi or the speed of light instead of using central sources).

**Comment by civanch** [comment]
I would agree with Fabio. If in a particular class m_pion (charged or neutral?) is needed ideally this constant may be defined in class constructor via HepPDT. 

Geant4 has also self consistent mass definition derived from PDG for all hadrons but it should not be used at RECO step.
